addons/binding/org.openhab.binding.yamahareceiver/README.md

```diff
@@ -25,52 +25,50 @@ select radio stations from a configured menu.
-`tuner_band#playback_channels` | `String` | Set the band (FM or DAB) for tuner input when device supports DAB (e.g. RX-S601D).
+The implemented channels for a zone thing are grouped in three groups. These are the zones supported: `Main_Zone`, `Zone_2`, `Zone_3`, `Zone_4`.
 
-Navigation control channels are:
+Zone control channels are:
 
 Channel | openHAB Type | Comment
 --------|------|------
-`navigation_menu#navigation_channels` | `String` | Select or display the full or relative path to an item.
-`navigation_current_item#navigation_channels` | `Number` | Get the current item of the current menu.
-`navigation_total_items#navigation_channels` | `Number` | Get the total count items in the current menu.
-`navigation_level#navigation_channels` | `Number` | Get the current menu level.
-`navigation_updown#navigation_channels` | | Move the cursor up or down.
-`navigation_leftright#navigation_channels` | | Move the cursor to the left or right.
-`navigation_select#navigation_channels` | | Select the current item.
-`navigation_back#navigation_channels` | | Navigate to the parent menu.
-`navigation_backtoroot#navigation_channels` | | Navigate back to the root menu.
+`zone_channels#power` | `Switch` | Switches the zone ON or OFF. Your receiver has to be in network standby for this to work.
+`zone_channels#mute` | `Switch` | Mute or Unmute the receiver.
+`zone_channels#volume` | `Dimmer` | Set's the receivers volume as percentage.
+`zone_channels#volumeDB` | `Number` | Set's the receivers volume in dB.
+`zone_channels#input` | `String` | Set's the input selection, depends on your receiver's real inputs. Examples: HDMI1, HDMI2, AV4, TUNER, NET RADIO, etc.
+`zone_channels#surroundProgram` | `String` | Set's the surround mode. Examples: `2ch Stereo`, `7ch Stereo`, `Hall in Munic`, `Straight`, `Surround Decoder`.
+`zone_channels#scene` | `String` | Sets the scene. Examples: `Scene 1`, `Scene 2`, `Scene 3`, `Scene 4`. Write only (state updates are not available). May not be supported on all models (e.g. RX-V3900).
+`zone_channels#dialogueLevel` | `Number` | Set's the receivers dialogue level. May not be supported on all models.
+`playback_channels#preset` | `Number` | Set a preset. Not supported by `Spotify` input. For `NET RADIO` input there is no way to get current preset (tested on RX-S601D, RX-V3900), so the preset is write only. For RX-V3900 the presets are alphanumeric `A1`,`A2`,`B1`,`B2` thus you need to use numbers grater than 100 that represent these presets as follows: 101, 102, 201, 202.
+`playback_channels#playback` | `String` | Set a play mode or get the current play mode. Values supported: `Previous`, `Play`, `Pause`, `Stop`, `Next`. Applies for inputs which support playback (`Spotify`, `SERVER`, `NET RADIO`, `Bluetooth`). Note that some values may not be supported on certain input type and AVR model combination. For `Spotify` and `Bluetooth` all values work, but for `NET RADIO` input only `Play` and `Stop` are supported (tested on RX-S601D).  
```

Comment:
Can you reformat the tables in this README with padding so they are easy to read? The [Markdown tables generator](https://www.tablesgenerator.com/markdown_tables) can help you with that.